massa-models/src/block_header.rs

```diff
@@ -97,8 +95,6 @@ impl SecuredHeader {
     }
 }
 
-impl SecureShareContent for BlockHeader {}
```

Comment:
I would have left the implementation here instead of moving it to denunciation.rs, because that's actually the way the ID of BlockHeader is computed

Reply:
moved